docs/design/design.md

```diff
@@ -76,10 +117,16 @@ There should be:
  - a client cert for the front proxy client (`front-proxy-client.crt`) using `front-proxy-ca.crt` as the CA with its key (`front-proxy-client.key`)
 
 
-If a given certificate and private key pair both exist, the generation step will be skipped and those files will be validated and used for the prescribed use-case.
-This means the user can, for example, prepopulate `/etc/kubernetes/pki/ca.{crt,key}` with an existing CA, which then will be used for signing the rest of the certs.
+Please note that:
 
-### Phase 2: Generate KubeConfig files for the master components
+1. If a given certificate and private key pair both exist, and its content is evaluated compliant with the above specs, the existing files will be used and the generation phase for the given certificate skipped.
+   This means the user can, for example, copy an existing CA to `/etc/kubernetes/pki/ca.{crt,key}` , and then  then `kubeadm` will use those files for signing the rest of the certs.
+2. Only for the CA, it is possible to provide the `ca.crt` file but not the `ca.crt` file, 
```

Comment:
but not the `ca.key` file

Reply:
add: _if all other certificates and kubeconfig files already are in place_
This condition will trigger a "External CA key" mode, which also implies the `csrsigner` controller in controller-manager won't be started

Reply:
Done, but in the code there there is no check for kubeconfig files :-(

Reply:
Oh yeah, could you send a PR to fix that?